raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -1677,4 +1698,14 @@ private void pollUntilSend(KafkaRaftClient client) throws InterruptedException {
         }, 5000, "Condition failed to be satisfied before timeout");
     }
 
+    private void deliverRequest(ApiMessage request) {
```

Comment:
What's the motivation for these two helper functions? We are still doing a one-line change here.

Reply:
It just simplifies the expressions a bit.